tensorflow/python/ops/array_ops.py

```diff
@@ -3926,6 +3926,77 @@ def reverse_sequence(input,
                      name=None,
                      seq_dim=None,
                      batch_dim=None):
+  """Reverses variable length slices.
+
+  This op first slices `input` along the dimension `batch_axis`, and for
+  each slice `i`, reverses the first `seq_lengths[i]` elements along the
+  dimension `seq_axis`.
+
+  The elements of `seq_lengths` must obey `seq_lengths[i] <=
+  input.dims[seq_dim]`, and `seq_lengths` must be a vector of length
+  `input.dims[batch_dim]`.
+
+  The output slice `i` along dimension `batch_axis` is then given by
+  input slice `i`, with the first `seq_lengths[i]` slices along
+  dimension `seq_axis` reversed.
+
+  For example:
+
+  ```python
```

Comment:
Can you rewrite this example as a doctest? (i.e. add `>>>` or `...` to the beginning of lines of code and put the result of each expression below it).